src/materials_processor/houdini/traverser.py

```
@@ -1,10 +1,12 @@
-"""Houdini material graph traversal."""
+import logging
+
+logger = logging.getLogger(__name__)
 
 try:
     import hou
 except:
```

Ruff (0.15.14)

[error] Do not use bare `except`

(E722)